pages/linux/sxiv.md

```diff
@@ -0,0 +1,24 @@
+# sxiv
+
+> Simple X Image Viewer.
+> More information: <https://github.com/muennich/sxiv>.
+
+- View an image:
+
+`sxiv {{path/to/file}}`
+
+- View an image in fullscreen mode:
+
+`sxiv -f {{path/to/file}}`
+
+- View an image or set of images, reading filenames from standard input:
+
+`echo {{path/to/file}} | sxiv -i`
+
+- View images as a slideshow:
+
+`sxiv -S {{delay}} {{path/to/image}} ...`
+
+- View images as thumbnails:
```

Comment:
```suggestion
- Open a space-separated list of images in thumbnails mode:
```

Reply:
Resolved in 9846d6a0e1471e8baad669860ac4c8c4092dbd5c